package.json

```diff
@@ -3,6 +3,8 @@
   "version": "20.0.0",
   "description": "A responsive and accessible date range picker component built with React",
   "main": "index.js",
+  "module": "esm/index.js",
+  "sideEffects": false,
```

Comment:
this is inaccurate; some of our files do have side effects.

Reply:
Thanks for pointing this out. I have just update to `initialize.js` and all css 
using Regex `^\w+\(` to find all self executed function. Hope I didn't miss any.